src/services/conseillers/controllers/deleteCandidatById.ts

```diff
@@ -237,6 +253,37 @@ const deleteCandidatById =
         .Model.accessibleBy(req.ability, action.read)
         .find(instructionSuppression);
 
+      const checkAccessMiseEnRelation = checkAccessReadRequestMisesEnRelation(
+        app,
+        req,
+      );
+
+      const misesEnRelation = await app
+        .service(service.misesEnRelation)
+        .Model.aggregate([
+          {
+            $match: instructionSuppressionMER,
+          },
+          { $match: checkAccessMiseEnRelation },
```

Comment:
c'est pas bon il faut faire un seul $match

Reply:
`$match: {
$and: [checkAccessMiseEnRelation],
... instructionSuppressionMER
}`

Reply:
quelque chose comme ça